students/barecki_andrzej/lesson_03_functions/L3_Exponentiation.py

```diff
@@ -0,0 +1,46 @@
+"""
+Given a positive real number a and a non-negative integer n. Calculate an without using loops, ** operator
+or the built in function math.pow(). Instead, use recursion and the relation an=a⋅an−1. Print the result.
+"""
+import os
+
+
+def input_validation(param_type):
+    global val
```

Comment:
Zerknij jak działa słowo kluczowe global: https://www.python-course.eu/python3_global_vs_local_variables.php
Nie ma potrzeby udostępniać tej zmiennej poza ciałem `input_validation`. 
Jeżeli chcesz sobie zadeklarować zmienną była widoczna przy `return` to zamiast `global val` zrób `val = None`. Zalecam poprawkę, przejrzyj pozostałe pliki pod tym kątem.